feat: 2023 (#232)
brown-ccv/ccv-website-content: blog/2023-year-in-review.md

    - Jordan Lawson
    - Joselynn Wallace
    - Paul Cao
    - Paul Xu
    - Anna Murphy
    - Ashley S. Lee
    - Bradford N. Roarr
    - Camilio Diaz
    - Carlos Paniagua
    - Ellen Duong
    - Ford McDonald
    - Galen Winsor
    - George Dang
    - Heather Yu
    - Isabel Restrepo
    - John Gerrard Holland
    - Joshua Lu
    - Robert Gemma, Jr.
    - Timothy Divoll
    - Paul Hall
    - Pratibha Raghunandan
    - Pritivi Thakur
    - David D Johnson
    - Geoffrey Avila
    - Nathan Wood
    - Paulo S Baptista
    - Singh Saluja
    - Prasad Bandarkar
    - Rain Fan
    - Yang Liu
    - Jarrod Winsor
    - Linnea Wolfe
    - Michael Antonelli
    - Paul Koussa
    - Paul Stey
    - Sam Fulcomer
    - Thomas Serre
    - Timothy Poli
date: 2024-01-11
slug: 2023-year-in-review
team: GSDC (Graphics, Software and Data Core)
project:
description: 2023 was a year of numerous strides at CCV and in the research community here at Brown. Here are some of the many highlights over the past year!
tags:
    - software

---
 
2023 was a year of numerous strides at CCV and in the research community here at Brown. Thank you for your continued participation and support. 

> Here are some quick statistics from our Brown community made in 2023. We had 1557 active Oscar users, which is a 24.66% increase in activity from the year prior. Over 3.6 million jobs were run in Oscar totalling over 57 million CPU core hours! Since 2018, the number of active oscar users has increased and well as demand for resources. Over **470** questions were asked and helped in our weekly office hours.

Reviewer: ```suggestion
Here are some quick statistics from our Brown community made in 2023. We had 1557 active Oscar users, which is a 24.66% increase in activity from the year prior. Over 3.6 million jobs were run in Oscar totalling over 57 million CPU core hours! Since 2018, the number of active oscar users has increased as well as demand for resources. Over **470** questions were asked and helped in our weekly office hours.
```

Alongside supporting Oscar for high-performance computing needs, CCVers provided academic support to the Brown community. We hosted 17 Data Science, Computing, and Visualization Workshops (DSCoVs) with the Data Science Initiative (DSI). Topics included Gene Annotation Resources in R, ChatGPT's API and Prompt Engineering, Building VR Application in Unity, and more! Jupyterhub, cloud-hosted Jupyter Notebooks for multiple users, supported *8* undergraduate and graduate courses. In June, we hosted a bootcamp, a series of online/hybrid tutorials, wherein *over 175* members of the Brown community signed up to learn about research computing resources and get hands-on practice using Oscar.

In 2023, we supported 60 projects and collaborations throughout Brown. The following are some highlights from those projects.

- We launched our first mobile applications: [**SOMA**](https://somatheapp.com/), an app focused on pain management, and [**MAPPS**](https://www.mappsproject.com/), an app designed to explore social interactions and their influence on disease transmission patterns.

> - [**Honeycomb**](https://brown-ccv.github.io/honeycomb-docs/), a template for reproducible psychological tasks for clinic, laboratory, and home use has undergone a significant upgrade and has released version 3.2.6.

Reviewer 2: @eldu here's my best sentence for Honeycomb! Not sure how much better it is haha
```suggestion
- [**Honeycomb**](https://brown-ccv.github.io/honeycomb-docs/) released versions 3.1 and 3.2 which better align the repository with [jsPsych](https://www.jspsych.org/7.3/) best practices.
```

- [**Hierarchical Sequential Sampling Modeling (HSSM)**](https://lnccbrown.github.io/HSSM/) —a contemporary Python toolbox integrating cutting-edge likelihood approximation methods within the Python Bayesian ecosystem—was released on the Python Package Index (PyPI) in late June.
- We developed and released two packages in Julia, a rising programming language in the data science space. [IceFlowTracker.jl](https://github.com/WilhelmusLab/IceFloeTracker.jl) was created in collaboration with the [Wilhelmus Lab](https://wilhelmuslab.me/) for tracking ice floes using moderate resolution imaging spectroradiometer (MODIS) data. [Chamber.jl](https://github.com/brown-ccv/Chamber.jl/tree/master) is a Julia package for simulating the eruption of a volcano using a model for the frequency of eruptions of upper crustal magma chambers based on Degruyter and Huber (2014).

> - We developed and released a versatile utility script to streamline data exports to Oscar, led two XNAT Workshops for the neuroimaging community, and contributed to the MNE-BIDS open-source project. Furthermore, we enhanced the xnat2bids pipeline to accommodate EEG and Physiological data, highlighting our collective commitment to optimizing neuroimaging workflows and fostering open-source collaboration.

Reviewer: ```suggestion
- In collaboration with [the Behavior and Neurodata Core (BNC)](https://bnc.brown.edu), we developed and released a versatile utility script to streamline data exports to Oscar, led two XNAT Workshops for the neuroimaging community, and contributed to the MNE-BIDS open-source project. Furthermore, we enhanced the XNAT-to-BIDS exporting pipeline to accommodate EEG and Physiological data, highlighting our collective commitment to optimizing neuroimaging workflows and fostering open-source collaboration.
```

- [**Honeycomb**](https://brown-ccv.github.io/honeycomb-docs/), a template for reproducible psychological tasks for clinic, laboratory, and home use has undergone a significant upgrade and has released version 3.2.6.
- [**Hierarchical Sequential Sampling Modeling (HSSM)**](https://lnccbrown.github.io/HSSM/) —a contemporary Python toolbox integrating cutting-edge likelihood approximation methods within the Python Bayesian ecosystem—was released on the Python Package Index (PyPI) in late June.
- We released two Julia Packages
- We developed and released a versatile utility script to streamline data exports to Oscar, led two XNAT Workshops for the neuroimaging community, and contributed to the MNE-BIDS open-source project. Furthermore, we enhanced the xnat2bids pipeline to accommodate EEG and Physiological data, highlighting our collective commitment to optimizing neuroimaging workflows and fostering open-source collaboration.

> 

Reviewer 3: ```suggestion
- We continued to support the local and state governments with advertising and getting collaborators for their public policy projects: [North Carolina Project Portal](https://projectportal.nc.gov), [Research Partnerships Portal of the City of San Antonio in Texas](https://researchpartnerships.sanantonio.gov)

```
